pkg/prepare/prepare.go

```diff
@@ -49,7 +49,7 @@ type Preparer interface {
 	ConfigureKubeDNS() error
 }
 
-// Ensure the Prepare fits the Preparer interface
+// Ensure the Prepare fits the Preparer interface.
 var _ Preparer = (*Prepare)(nil)
```

Comment:
For use via the EE integration

Reply:
For use in other projects. They use the interface instead of having to worry about the behind-the scenes code.

Reply:
I'm not sure that the `Preparer` interface is needed because the `Preparer` is always used as-is by the commands and never in a context where we could offer an extension point.

Reply:
I implemented it so that if we modified the Preparer object, that we would not break any integrations with other projects, as they would use the interface.

Reply:
I'm not sure to get the point but nevermind that's out of the PR scope. For me, it's the same as using the public methods from the struct, which defines the API. Interfaces express behaviour which should be defined on the [consumer side](https://github.com/golang/go/wiki/CodeReviewComments#interfaces).

Reply:
Ah now I get it. Yes, you are correct, we can remove this in another issue.

Reply:
https://github.com/containous/maesh/issues/583